pkg/importer/data-processor.go

```diff
@@ -308,6 +308,8 @@ func (dp *DataProcessor) resize() (ProcessingPhase, error) {
 			return ProcessingPhaseError, err
 		}
 		dp.preallocationApplied = dp.preallocation
+	} else {
```

Comment:
It's cleaner to leave `dp.preallocationApplied = dp.preallocation` outside the condition and do it once, but it's not a must.

/test pull-containerized-data-importer-e2e-ceph

Reply:
Good point in my mind it was not logically equivalent because the stuff above could return a different phase, but that would not change outside the else.